packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -2,18 +2,30 @@
 // Licensed under the MIT license.
 
 import { ChatClient, ChatParticipant, ChatThreadClient } from '@azure/communication-chat';
-import { CommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
+import {
+  CommunicationTokenCredential,
+  CommunicationUserIdentifier,
+  getIdentifierKind
+} from '@azure/communication-common';
 import { CommunicationIdentifier } from '@azure/communication-signaling';
 import { _createStatefulChatClientWithDeps } from '@internal/chat-stateful-client';
 import { _IdentifierProvider } from '@internal/react-components';
 import { nanoid } from 'nanoid';
 import React, { useEffect, useState } from 'react';
-import { ChatAdapter, ChatComposite, createAzureCommunicationChatAdapterFromClient } from '../../../../src';
+import {
+  ChatAdapter,
+  ChatComposite,
+  createAzureCommunicationChatAdapterFromClient,
+  FileDownloadError,
+  FileDownloadHandler
+} from '../../../../src';
```

Comment:
nit: Try avoid imports direct from the src's index.ts as it can cause circular import chains